homeassistant/helpers/device_registry.py

```diff
@@ -244,11 +244,10 @@ class DeviceEntry:
     """Device Registry Entry."""
 
     area_id: str | None = attr.ib(default=None)
-    config_entries: set[str] = attr.ib(converter=set, factory=set)
+    config_entries: list[str] = attr.ib(factory=list)
```

Comment:
Isn't this change a breaking change?

Reply:
PR: https://github.com/home-assistant/core/pull/119959